### PR TITLE
Add missing API functions to the rst documentation

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**

### CODE REVIEW

File: docs/api/connection.rst [hunk 1]
```diff
@@ -418,7 +418,8 @@ Functions
 
 .. c:function:: drizzle_return_t drizzle_close(drizzle_st *con)
 
-   Gracefully disconnect from a server (leaves the connection object available for a reconnect
+   Gracefully disconnect from a server (leaves the connection object available
+   for a reconnect
```
Comment (ben-palmer-sociomantic) [inline docs/api/connection.rst]: Missing an ending `)`.

**Review by ben-palmer-sociomantic — COMMENTED** [review]
One tiny comment then LGTM

File: docs/api/connection.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply docs/api/connection.rst]: Ah true, fixed

**Comment by ben-palmer-sociomantic** [comment]
ok LGTM.

**Pull request merged**